scripts/view-test-artifacts.ts

```diff
@@ -115,7 +115,8 @@ const downloadSingleTestArtifact = async (tempDir: string, artifact: Required<Te
 
   return Promise.all(
     listObjectsResponse.Contents.filter(hasKey).map(({ Key }) => {
-      const filePath = path.join(artifactDownloadPath, Key.split('/').pop()!);
+      const filePath = path.join(artifactDownloadPath, Key.split('/').slice(3).join('/'));
```

Comment:
This change is to support the artifact change, which have the run index (0 & 1) in the path